test/100-bridge-iptables.bats

```diff
@@ -321,3 +322,16 @@ EOF
     expected_rc=1 run_netavark setup $(get_container_netns_path) <<<"$config"
     assert_json ".error" "unsupported ipam driver someDriver" "Driver is not supported error"
 }
+
+@test "$fw_driver - isolate networks" {
+    run_netavark --file ${TESTSDIR}/testfiles/simplebridge.json setup $(get_container_netns_path)
+    result1="$output"
+    assert_json "$result1" 'has("podman")' == "true" "object key exists"
+
+    run_netavark --file ${TESTSDIR}/testfiles/connectbridge.json setup $(get_container_netns_path)
+    result2="$output"
+    assert_json "$result2" 'has("podman1")' == "true" "object key exists"
+
+    # This should fail, right?
+    run_in_container_netns ping -c 1 10.88.0.2
+}
```

Comment:
This test does not do what it should.
You setup two networks on one container. That should always work. We need to test that we cannot communicate between two containers on different subnets but still make sure that it works on the same network. The test infra currently does not expose a second container but you should be able to add a second namespace manually.